### ISSUE TITLE
Issues with County Council geographic data from cso_get_geo()

### ISSUE THREAD

**Issue by jamesor19** [CSOIreland/csodata#5]
Some Issue with the County Council geographic data from cso_get_geo(). 

The following code:
`cso_get_geo_meta(cso_get_geo("cc"))`

when run results in the following error

```
Error in wk_handle.wk_wkb(wkb, s2_geography_writer(oriented = oriented,  : 
  Loop 50 is not valid: Edge 1391 has duplicate vertex with edge 1394
```
Unsure if issue with cso_get_geo_meta() or the sf dataframe itself. Will need to be investigated.

**Comment by jamesor19** [comment]
Will be resolved in #9

**Comment by jamesor19** [comment]
Fix added in #9.
